tests/conftest.py

```diff
@@ -0,0 +1,85 @@
+"""Pytest conftest.py"""
+
+import json
+from pathlib import Path
+from typing import Any, Dict
+from unittest.mock import MagicMock
+
+import pytest
+
+TESTS_DIR = Path(__file__).parent
+JSON_DIR = TESTS_DIR / "json"
+EMPTY_PAYLOAD: Dict[str, Any] = {}
+
+
+def load_json_payload(filename: str) -> Dict[str, Any]:
+    with open(JSON_DIR / filename, encoding="utf-8") as file:
+        return json.load(file)
+
+
+@pytest.fixture
+def github_payload() -> Dict[str, Any]:
+    """Return GitHub payload as a Python dict."""
+    try:
+        file_path = JSON_DIR / "github-payload.json"
+        with open(file_path, encoding="utf-8") as file:
+            return json.load(file)
+    except FileNotFoundError:
```

Comment:
Under what circumstances can this happen? The `file_path` isn't affected by any arguments and is always the same.

Reply:
I'll remove this to make it simpler. I was probably just being overzealous.